spec/integration/init_web_spec.cr

```diff
@@ -68,6 +68,28 @@ describe "Initializing a new web project" do
     message = "Folder named test-project already exists, please use a different name"
     output.to_s.strip.should contain(message)
   end
+
+  it "does not create project if project name is not a valid directory name" do
+    output = IO::Memory.new
+    Process.run(
+      "crystal src/lucky.cr init 'test project'",
+      output: output,
+      shell: true
+    )
+    message = "Directory name cannot contain spaces or special characters."
+    output.to_s.strip.should contain(message)
+  end
+
+  it "does not create project if project name is invalid" do
+    output = IO::Memory.new
+    Process.run(
+      "crystal src/lucky.cr init 'test *roject'",
+      output: output,
+      shell: true
+    )
```

Comment:
I added a module that might help with this https://github.com/luckyframework/lucky_cli/blob/master/spec/support/should_run_successfully.cr

So I think you could do `should_run_successfully "crystal src/lucky.cr init 'test *roject'"`, io: output`